### PR TITLE
test: e2e automation tests fix

### PR DESCRIPTION
**Pull request by Yakitrak**
## Describe your changes

Ticket: https://pins-ds.atlassian.net/browse/ASB-1879

Update packages/e2e-tests

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js [hunk 1]
```diff
@@ -54,9 +54,10 @@ export class PO_ExaminationTimetable {
 			cy.wrap(answers)
 				.each((ans) => myAnswers.push(ans.text().trim()))
 				.then(() => {
-					console.log(myAnswers);
 					myAnswers.shift();
+					console.log({ expectedAnswers, myAnswers });
 					expect(expectedAnswers).to.deep.eq(myAnswers);
+					//expect(expectedAnswersFinal).to.deep.eq(myAnswers);
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:60]: Can remove commented line

File: packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import { PO_ProjectPage } from '../pageObject/Search-and-project-pages/PO_ProjectPage';
+
+const examinationTimetable = new PO_ExaminationTimetable();
+const projectSearch = new PO_ProjectSearch();
+const projectPage = new PO_ProjectPage();
+
+describe('User registers as themselves to have their say against the examination timetable', () => {
+	it('Navigates to the examination timetable page for a project and start the journey', () => {
+		cy.clearCookies();
+		cy.visit('/projects/EN010120/examination-timetable');
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:12]: This test assumes EN010120 will be at stage Examination and have an open deadline in any env we run it.
Similar for WA020002 being in registration.  

Something to think about for the future, we could introduce a variable to the test EXAMINATION_CASE_REFERENCE.

File: packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
+import { PO_ProjectSearch } from '../pageObject/Search-and-project-pages/PO_ProjectSearch';
+import { PO_ProjectPage } from '../pageObject/Search-and-project-pages/PO_ProjectPage';
+
+const examinationTimetable = new PO_ExaminationTimetable();
+const projectSearch = new PO_ProjectSearch();
+const projectPage = new PO_ProjectPage();
+
+describe('User registers as themselves to have their say against the examination timetable', () => {
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:9]: User registers as themselves to have their say against the examination timetable

Should probably be:

User makes a submission against a examination timetable deadline item

File: packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
```
Comment (Philip-Philippou1) [inline packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:1]: This file should perhaps be called examination-timetable-submission-....
Register confuses with the other journey.

Also, should we be creating sub directories here to group the tests , the filenames could then be shorter and more specific.

Comment (JacobRMullally) [reply packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:1]: Renamed as examination-timetable-myself-submission

File: packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js (hunk 3 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:9]: Updated to 'User makes a submission against a examination timetable deadline item'

File: packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js (hunk 2 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js:12]: Will invetigate how we make configurable in future work

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js (hunk 1 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:60]: Have removed this now

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**